packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -232,9 +233,14 @@ export async function createTemporaryTsConfig(includedPackageDirs: string[]) {
     extends: './tsconfig.json',
     include: [
       // These two contain global definitions that are needed for stable API report generation
-      'packages/cli/asset-types/asset-types.d.ts',
+      resolvePackagePathBackend(
+        '@backstage/cli',
+        'asset-types/asset-types.d.ts',
+      ),
       ...includedPackageDirs.map(dir => join(dir, 'src')),
     ],
+    // we don't exclude node_modules so that we can use the asset-types.d.ts file
+    exclude: [],
```

Comment:
Is this required, i.e. does it work if you remove it? Afaik TS doesn't really care whether something is included/excluded if it's referenced directly. Then again I see that `node_modules` being excluded would prevent it from being included.

Reply:
Yeah indeed `node_modules` being excluded prevents the file from being included. @mtlewis helped with debugging here, because we couldn't get it to work 😆